accounts/abi/bind/backends/simulated.go

```diff
@@ -284,6 +284,25 @@ func (b *SimulatedBackend) SendTransaction(ctx context.Context, tx *types.Transa
 	return nil
 }
 
+// JumpTimeInSeconds adds skip seconds to the clock
+func (b *SimulatedBackend) JumpTimeInSeconds(skip int64) error {
+	if skip < 0 {
```

Comment:
You can accept an unsigned integer and remove this check.

Reply:
Why not use `time.Duration` instead of an untyped int with a convoluted method name explaining it's in seconds?

Reply:
Good idea...

Reply:
Done :-)